src/Modules/OAuth/README.md

```diff
@@ -0,0 +1,62 @@
+# OAuth Module
+The oAuth module provides a way to authenticate users using Microsoft Azure.
```

Comment:
Juiste schrijfwijze os OAuth

Is toch meer mogelijk dan Azure? Maar voorlopig is enkel Azure geïmplementeerd. Juist?
Indien ja: 
> The oAuth module provides a way to authenticate users using OAuth instead of email and password. At this moment only Microsoft Azure is implemented.

En dan een title `## Azure` en al de rest van de titels juist zetten